nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-controller-service/pom.xml

```diff
@@ -88,12 +88,6 @@
             <artifactId>kudu-client</artifactId>
             <version>${kudu.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.apache.kudu</groupId>
```

Comment:
Why was this removed?

Reply:
See the comments summarizing the removals.